CardParts/src/Extensions/UIColor.swift

```diff
@@ -11,6 +11,8 @@ extension UIColor {
     static var turboGenericGreyTextColor : UIColor {get{return UIColor.color(169, green: 169, blue: 169)}}
     static var turboCardPartTitleColor : UIColor {get{return UIColor.color(17, green: 17, blue: 17)}}
     static var turboCardPartTextColor : UIColor {get{return UIColor.color(136, green: 136, blue: 136)}}
+    static var turboCardPartAttributedTextColor : UIColor {get{return UIColor.color(0, green: 0, blue: 255)}}
```

Comment:
just rename it to `CardPartAttributedTextColor`  remove Turbo from it , at some point we will be removing turbo references in variable names.